data-prepper-plugins/csv-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/csv/CsvProcessorConfig.java

```diff
@@ -57,7 +57,7 @@ public class CsvProcessorConfig {
     private List<String> columnNames;
 
     @JsonProperty("csv_when")
-    @JsonPropertyDescription("Allows you to specify a [conditional expression](https://opensearch.org/docs/latest/data-prepper/pipelines/expression-syntax/), " +
```

Comment:
I might be wrong but I thought we want to keep this markdown format here so it will be rendered in the documentation website. @chenqi0805

Reply:
We aren't rendering the documentation website from this.

Reply:
Not today, but we are planning to do that, see this: https://github.com/opensearch-project/documentation-website/pull/7651

Reply:
Right. We plan to generate the config table from those annotations